src/core/a-entity.js

```diff
@@ -312,6 +312,7 @@ var proto = Object.create(ANode.prototype, {
       pauseComponent(component, this.sceneEl);
       component.remove();
       delete this.components[name];
+      this.emit('componentremoved', { name: name });
```

Comment:
would it already be available as `event.target`? that seems like enough if so, otherwise maybe helpful to add it.